init-odkx-sync-endpoint.py

```diff
@@ -148,7 +148,7 @@ def run_docker_builds():
 
 
 def run_sync_endpoint_build():
-    os.system("git clone -b master --single-branch --depth=1 https://github.com/odk-x/sync-endpoint ; \
+    os.system("git clone --single-branch --depth=1 https://github.com/odk-x/sync-endpoint ; \
```

Comment:
if you don't specify the branch, won't it default to master?

Reply:
It will default to the branch defined as "default branch" under the settings tab in the repo in GH... (so currently yes; master. But then we can easily change it to main later)

Reply:
but don't we want it pointing at circleci-taketwo?